packages/landmarks/landmarks.1.4/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "A simple profiling library"
+description: """
+Landmarks is a simple profiling library for OCaml. It provides
+primitives to measure time spent in portion of instrumented code. The
+instrumentation of the code may either done by hand, automatically or
+semi-automatically using the ppx pepreprocessor (see landmarks-ppx package).
+"""
+maintainer: ["Marc Lasson <marc.lasson@lexifi.com>"]
+authors: ["Marc Lasson <marc.lasson@lexifi.com>"]
+license: "MIT"
+homepage: "https://github.com/LexiFi/landmarks"
+bug-reports: "https://github.com/LexiFi/landmarks/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/LexiFi/landmarks.git"
+available: arch = "x86_64" | arch = "x86_32"
```

Comment:
is the available field required on both package ?
I was hoping it wasn't because the "ppx" package depends on the runtime.